ee/zentral/contrib/intune/management/commands/intune_sync.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from django.core.management.base import BaseCommand, CommandError
+from zentral.contrib.intune.api_client import Client
+from zentral.contrib.intune.models import Tenant
+from zentral.contrib.intune.tasks import do_sync_inventory
+
+logger = logging.getLogger("zentral.contrib.intune.management.commands.intune_sync")
+
+
+class Command(BaseCommand):
+    help = "Synchronize MS Intune Inventory"
+
+    def add_arguments(self, parser):
+        parser.add_argument('--list-tenants', action='store_true', dest='list_tenants', default=False,
+                            help='list MS Intune Tenants')
+        parser.add_argument('--tenant', dest='tenant_id', help='MS Intune Tenant ID')
+
+    def handle(self, *args, **kwargs):
+        if kwargs.get("list_tenants"):
+            for tenant in Tenant.objects.all().order_by("name"):
+                print("Name:", tenant.name, "UUID:", tenant.tenant_id)
+            return
+        tenant_id = kwargs.get("tenant_id")
+        if not tenant_id:
+            return
```

Comment:
Maybe an error message here.

Reply:
Same happens on [wsone_sync.py](https://github.com/zentralopensource/zentral/blob/6e63568a1ebf9a77b01f6cc04a96e68509a7cac4/ee/zentral/contrib/wsone/management/commands/wsone_sync.py#LL26C18-L26C18)

I'll `raise` an error here and another PR for WSOne.